pkgs/top-level/all-packages.nix

```diff
@@ -29412,6 +29412,8 @@ with pkgs;
 
   ia-writer-duospace = callPackage ../data/fonts/ia-writer-duospace { };
 
+  ia-writer-quattro = callPackage ../data/fonts/ia-writer-quattro { };
+
```

Comment:
New top-level packages should go to `pkgs/by-name` https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md
```suggestion
```